README.md

```diff
@@ -332,10 +336,10 @@ The `--bl-settings-version` depends on the SDK version used. Check the following
 
 SDK Version   | BL Settings Version
 ------------- | -------------------
-12.0 - 15.2.0 | 1
+12.0 -        | 1
 15.3.0 -      | 2
 
-The Bootloader DFU settings version supported and used by the SDK you are using can be found in `nrf_dfu_types.h` in the `bootloader` library.
+The Bootloader DFU settings version supported and used by the SDK you are using can be found in `nrf_dfu_types.h` in the `bootloader` library. Bootloaders compiled from nRF5 SDK 15.3.0 and higher also supports BL Settings Version 1, depending on the project settings. If using version 2, boot validation for SD and APP can be generated with the settings page using the `--sd-boot-validation` and `--app-boot-validation` in the same way as for DFU packages.
```

Comment:
It might be worth it to specify that new bootloaders use only version 2, but it should handle version 1 fine (if configured to). If it boots with a settings page version 1 it will translate it to version 2 before continuing.

Reply:
Added